.github/workflows/run-clang-format.sh

```diff
@@ -12,19 +12,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+FILE_LIST_OUT=${TMPDIR:-/tmp}/file-list.$$.txt
+FORMAT_OUT=${TMPDIR:-/tmp}/clang-format-diff.$$.out
 
-FORMAT_OUT=${TMPDIR:-/tmp}/clang-format-diff.out
+function cleanup {
+  rm -f ${FORMAT_OUT} ${FORMAT_LIST_OUT}
+}
+trap cleanup EXIT
 
 # Run on all the files that are affected
-clang-format -i --style=file $(git diff --name-only --diff-filter=AM -r origin/master | grep '\(\.cc\|\.h\)$') 2> /dev/null
 
-# Check if we got any diff
-git diff > ${FORMAT_OUT}
+# Finding all affected files and list them for inspection in the log output
+git diff --name-only --diff-filter=AM -r origin/master | grep '\(\.cc\|\.h\)$' | tee "${FILE_LIST_OUT}"
```

Comment:
Take a look at `GITHUB_BASE_REF` and `GITHUB_HEAD_REF` -- see the full amount at https://docs.github.com/en/actions/reference/environment-variables